modules/persistent-storage-csi-gcp-pd-encrypted-pv.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+[id="persistent-storage-csi-gcp-pd-encrypted-pv_{context}"]
+= Creating a custom-encrypted persistent volume
+
+When you create a `PersistentVolume` object, {product-title} creates a persistent volume (PV). You can add a custom encryption key in Google Cloud Platform (GCP) to protect PVs in your cluster by encrypting a newly created PV.
+
+You can create an attached volume that uses customer-managed encryption keys (CMEK) on a new or existing cluster by using a new or existing Cloud KMS key.
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+* You have created a PVC using a the GCP PD CSI driver.
+* You have created a GCP Cloud KMS key ring and key version.
+
+For more information about CMEK and Cloud KMS resources, see link:https://cloud.google.com/kubernetes-engine/docs/how-to/using-cmek[Using customer-managed encryption keys (CMEK)].
+
+.Procedure
+To create a custom-encrypted PV, complete the following steps:
+
+. Create a storage class with the Cloud KMS key. The following example enables dynamic provisioning of encrypted volumes:
++
+[source,yaml]
+--
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-gce-pd-cmek
+provisioner: pd.csi.storage.gke.io
+volumeBindingMode: "WaitForFirstConsumer"
+allowVolumeExpansion: true
+parameters:
+  type: pd-standard
+  disk-encryption-kms-key: projects/key-project-id/locations/location/keyRings/key-ring/cryptoKeys/key <1>
```

Comment:
Please emphasize what parts of the key ID must be provided by the user, e.g. `projects/<key-project-id>/locations/<location>/keyRings/<key-ring>/cryptoKeys/<key>`. Link(s) to Google docs could be helpful: https://cloud.google.com/kms/docs/resource-hierarchy#retrieve_resource_id https://cloud.google.com/kms/docs/getting-resource-ids

Reply:
Thanks, I have wrapped user-provided values in `<>` and added the google links to the annotation note description.